test/integration/http/tap_handler.go

```diff
@@ -113,6 +114,9 @@ func (t *tapHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(500)
 		return
 	}
+	if t.statusCode > 299 {
+		w.WriteHeader(t.statusCode)
```

Comment:
I forgot to add a return here which I think is what is causing the "superfluous" log line that you see in the tests.  Do you want to just add that?

Reply:
Will do!